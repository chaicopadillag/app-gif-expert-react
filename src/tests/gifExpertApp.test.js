import React from 'react';
const { shallow } = require('enzyme');
const { default: GitExpertApp } = require('../GitExpertApp');

describe('Test de <GifExpertApp/>', () => {
	test('Debe hace Match con SnatShop', () => {
		const wraper = shallow(<GitExpertApp />);
		expect(wraper).toMatchSnapshot();
	});
	test('Debe hacer to match la cantidad de veces que envio categorias', () => {
		const cats = ['Terror', 'Comedia', 'Drama'];
		const wraper = shallow(<GitExpertApp defaultCatg={cats} />);
		expect(wraper).toMatchSnapshot();
		expect(wraper.find('GridGif').length).toBe(cats.length);
	});
});
