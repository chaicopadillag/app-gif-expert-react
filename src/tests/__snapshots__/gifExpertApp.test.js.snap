// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test de <GifExpertApp/> Debe hace Match con SnatShop 1`] = `
<Fragment>
  <h2>
    Git Expert App
  </h2>
  <AddCategory
    setCategorias={[Function]}
  />
  <hr />
  <div />
</Fragment>
`;

exports[`Test de <GifExpertApp/> Debe hacer to match la cantidad de veces que envio categorias 1`] = `
<Fragment>
  <h2>
    Git Expert App
  </h2>
  <AddCategory
    setCategorias={[Function]}
  />
  <hr />
  <div>
    <GridGif
      categoria="Terror"
      key="Terror"
    />
    <GridGif
      categoria="Comedia"
      key="Comedia"
    />
    <GridGif
      categoria="Drama"
      key="Drama"
    />
  </div>
</Fragment>
`;
